src/app/Providers.tsx

```diff
@@ -0,0 +1,13 @@
+"use client";
```

Comment:
what does use client do?

Reply:
By default, Next.js supports both server-side and client-side rendering, and you can access and manipulate DOM elements using standard React methods. The next/dynamic package allows for dynamic switching between server-side and client-side rendering if you have specific requirements that require client-side execution. **That line make it run in the client side to read the user currently dark/light themes.**